README.md

```diff
@@ -1,15 +1,25 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/FgMJElkj)
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+    1. Asymptotic analysis doesn't take into account constant factors. So while the constant factors do take time to run or memory to store, they aren't analysed. 
+    2. Asymptotic analysis looks at the growth rate as the input increases, but for small input sizes one algorithm might be better than another even though it's asymptotic analysis suggests otherwise. For example some implementations of merge/quick sort use insertion sort for small arrays. They do this because the growth rate of $n^2$ grows slower than $lg(n)$ with small inputs, but as n increases $lg(n)$ over takes $n^2$
```

Comment:
Not true. $\log(n)$ always grows slower than $n^2$. Case in point: $n=2$; $\log_2(n) = 1$, $n^2 = 4$.